### ISSUE TITLE
Invisible Texture / Colour

### ISSUE THREAD

**Issue by JustMe77** [Pottus/Texture-Studio#47]
Well, I'd suggest to create a command to set an object's texture / color invisible.
I know it's already possible, but sometimes it's hard to remember the texture / color, so why not just making a simple command?

Something like:

/mtinvisible <index id> will set the index from the object you're currently editing to invisible.
(Maybe a better cmd name :D)

Should be very simple :)

**Comment by Crayder** [comment]
Well, there are many fully transparent textures. They all have different effects, like object flags. Some let you see through objects behind them, some don't... which does what depends on the object you're editing. It's weird and hard to explain.

If I do do this, it'll be with SetObjectMaterialText, which could make text editing slightly more complicated.

**Comment by JustMe77** [comment]
You could use instead a texture the color which makes the object invisible, that would work always 100 % without any problems.

**Comment by Crayder** [comment]
>that would work always 100 % without any problems.
False, it to would cause the same things I was talking about in my last message.
